src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -577,6 +582,14 @@ public SscsHearingRecordingCaseData getSscsHearingRecordingCaseData() {
         }
     }
 
+    @JsonIgnore
+    public PostponementRequest getPostponementRequest() {
+        if (postponementRequest == null) {
```

Comment:
Do you need this getter? Can you not just use the default one pulled in from lombok?

Reply:
just to facilitate handling the object when null, following the examples from other objects in the code including the one you pointed out I should avoid bursting the limit of the attributes in the class.